packages/app/README.md

```
@@ -0,0 +1,119 @@
+pnpm build            # production build
+pnpm preview          # preview production build locally
+pnpm typecheck        # vue-tsc type check
+pnpm lint             # oxlint + eslint (check only)
+pnpm lint:fix         # oxlint + eslint --fix (always use this; never fix manually)
+pnpm test             # vitest watch mode
+pnpm coverage         # vitest run --coverage
+```
+
+### App Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)